src/components/Radio/Radio.js

```diff
@@ -80,14 +82,20 @@ export default class Radio extends React.Component {
       </div> : '';
 
     const fieldLabel = this.props.fieldLabel ?
-      <div className={`${LABEL_CLASSES.LABEL}`}>
-        {this.props.fieldLabel}
-        {message}
-        {info}
-      </div> : '';
+        <div className={`${LABEL_CLASSES.LABEL}`}>
```

Comment:
You can do this with similar cases.

```suggestion
        <div className={LABEL_CLASSES.LABEL}>
```